code/game/objects/items/storage/pillbottles.dm

```diff
@@ -0,0 +1,231 @@
+
+/*
+ * Pill Bottles
+ */
+/obj/item/storage/pill_bottle
+	name = "pill bottle"
+	desc = "It's an airtight container for storing medication."
+	icon_state = "pill_canister"
+	icon = 'icons/obj/medical/chemical.dmi'
+	inhand_icon_state = "contsolid"
+	worn_icon_state = "nothing"
+	lefthand_file = 'icons/mob/inhands/equipment/medical_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/equipment/medical_righthand.dmi'
+	w_class = WEIGHT_CLASS_SMALL
+	pickup_sound = 'sound/items/handling/pill_bottle_pickup.ogg'
+	drop_sound = 'sound/items/handling/pill_bottle_place.ogg'
+	storage_type = /datum/storage/pillbottle
+
+/obj/item/storage/pill_bottle/suicide_act(mob/living/user)
+	user.visible_message(span_suicide("[user] is trying to get the cap off [src]! It looks like [user.p_theyre()] trying to commit suicide!"))
+	return TOXLOSS
+
+/obj/item/storage/pill_bottle/multiver
+	name = "bottle of multiver pills"
+	desc = "Contains pills used to counter toxins."
+
+/obj/item/storage/pill_bottle/multiver/PopulateContents()
+	for(var/i in 1 to 7)
+		new /obj/item/reagent_containers/applicator/pill/multiver(src)
```

Comment:
Should probably be done in a separate PR, so ignore this, but we'd benefit from having these both be variables instead of overriding the proc.

Reply:
I have a part 4 coming up after this so i'll address this there